lib/python/picongpu/extra/utils/__init__.py

```diff
@@ -2,5 +2,6 @@
 from .memory_calculator import MemoryCalculator
 from .field_ionization import FieldIonization
 from . import FLYonPICRateCalculationReference
+from picmi_density_reader import cpp_fct_reader
 
-__all__ = ["FindTime", "MemoryCalculator", "FieldIonization", "FLYonPICRateCalculationReference"]
+__all__ = ["FindTime", "MemoryCalculator", "FieldIonization", "FLYonPICRateCalculationReference", "cpp_fct_reader"]
```

Comment:
```suggestion
__all__ = ["FindTime", "MemoryCalculator", "FieldIonization", "FLYonPICRateCalculationReference", "CppFctReader"]
```
Doesn't follow naming scheme.

Reply:
Personally I also prefer `Fn` for functions instead of `Fct`